scripts/test.py

```diff
@@ -158,12 +157,10 @@ def run_emsdk(cmd):
 
 print('test tot-upstream')
 run_emsdk('install tot-upstream')
-assert not os.path.exists(LIBC)
```

Comment:
why can't we check this any more?

Reply:
Sorry I should have explained.  This sequence of assert was checking the "activate" re-installs libraries in the cache.

However, in the now model there is no "install into cache step".   The cache is just part of the tar archive.. which means if you want to restore the cache after a `--clear-cache` you need to for a re-install from the tar archive (or just let it get built on demand).